packages/buckram/assets/styles/components/media/_interactive-content.scss

```diff
@@ -11,6 +11,7 @@
   }
   margin-bottom: if-map-get($interactive-content-margin-bottom, $type);
   padding: if-map-get($interactive-content-padding-top, $type) if-map-get($interactive-content-padding-right, $type) if-map-get($interactive-content-padding-bottom, $type) if-map-get($interactive-content-padding-left, $type);
+  text-indent: if-map-get($interactive-content-text-indent, $type);
```

Comment:
This doesn't actually fix the issue; the image was displayed inline so the indent was applied to it as if it were a paragraph.